### DISCUSSION TITLE
How to use <Image> with <GatsbyImage> and <StaticImage>?

### DISCUSSION THREAD

**Discussion by tauren** [chakra-ui/chakra-ui#4583] [Q&A]
I'm using Gatsby's `<GatsbyImage>` and `<StaticImage>` and am wondering if there is any way to make them work with Chakra's `<Image>`. The Chakra Image convenience props would be nice to have and make element styling consistent throughout my app. I tried `<Image as={StaticImage}`, but unfortunately nothing renders.

**Comment by tauren** [comment]
Upon further examination, even this doesn't work:

```
const CoverImage = styled(StaticImage)`
  border-radius: 8px;
`
```

This might be an issue with gatsby-plugin-image.

**Comment by trkohler** [comment]
having the same issue.
